data/free worlds intro.txt

```diff
@@ -1602,7 +1602,29 @@ mission "FW Bounty 1"
 		payment 200000
 		dialog `The militia officer who asked you to hunt down the "Rat Pack" arrives at your ship soon after you land and thanks you profusely. "Now we won't have to worry about them bothering ships in this sector anymore." She pays you <payment> and says she'll tell other militia leaders how helpful you were.`
 	on fail
-		dialog `You abandon hunting the "Rat Pack" since you are now a member of the Free Worlds militia.`
+		conversation
+			branch "Free Worlds"
+				or
+					has "Rescue Katya 1: active"
+					has "joined the free worlds"
+				
+			`You receive a sealed message from a courier who bears a Free Worlds insignia. You open it and read:`
+			``
+			`Captain <last>:`
+			`	Since you have decided to actively oppose the Free Worlds and our independence, we must retract the reward being offered to hunt the "Rat Pack" pirates. It disappoints us that you have chosen this path, but as members of the Free Worlds, we must respect your decision as a free individual.`
+			`	However, let this message also serve as a notification: do not enter Free Worlds territory without permission, or you will be fired upon.`
```

Comment:
Very true. But with 0 details and 0 timeline for Republic/Syndicate/Other faction plots, all I have to work with is "there is conflict at the time of faction alignment."